$NetBSD: patch-aa,v 1.2 2011/09/12 04:46:54 dholland Exp $

- quick and dirty LP64 fix: "unsigned" is more often 32-bit than
"unsigned long" is.

- disable inline unconditionally as the code in the package completely
misuses it.

--- defs.h.orig	1993-05-31 15:17:00.000000000 +0000
+++ defs.h
@@ -20,7 +20,7 @@
 typedef unsigned char		BYTE;
 typedef unsigned short int	WORD;	/* 16-bit unsigned on most systems */
 #ifndef _OS2EMX_H
-typedef unsigned long int	LONG;	/* 32-bit unsigned on most systems */
+typedef unsigned int		LONG;	/* 32-bit unsigned on most systems */
 #endif /* _OS2EMX.H */
 
 typedef int		BOOLEAN;
@@ -43,7 +43,7 @@ typedef int		BOOLEAN;
 
 /* 'inline' patch for compilers which can't handle this */
 
-#if !( defined( __CPLUSPLUS__ ) || defined( __cplusplus ) ) || defined( __TSC__ )
+#if 1
   #define inline
 #endif /* !( __CPLUSPLUS__ || __cplusplus ) */
 
